### PR TITLE
feat(code-runner): Add `molten-nvim`

### LINKED ISSUE #1027: [Feature request]: Add molten-nvim
### Is your feature related to a problem?

Please add this plugin [molten-nvim](https://github.com/benlubas/molten-nvim)

### Describe the new feature

image

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Uzaaft**
Closes #1027

### CODE REVIEW

File: lua/astrocommunity/code-runner/molten-nvim/init.lua
Comment (mehalter) [inline lua/astrocommunity/code-runner/molten-nvim/init.lua:36]:
```suggestion
```

this isn't necessary

File: lua/astrocommunity/code-runner/molten-nvim/init.lua
Comment (mehalter) [inline lua/astrocommunity/code-runner/molten-nvim/init.lua:29]:
```suggestion
  version = "^1", -- use version <2.0.0 to avoid breaking changes
```

unbelievably minor nit, take it or leave it haha

**Review by mehalter — CHANGES_REQUESTED** [review]
This looks great, only thing missing are the small nitpicks I put in and then this should set a bunch of keybindings to make it useful

File: lua/astrocommunity/code-runner/molten-nvim/init.lua
Comment (Uzaaft) [reply lua/astrocommunity/code-runner/molten-nvim/init.lua:36]: Why not? It's an optional dependency of the library?

Comment (mehalter) [reply lua/astrocommunity/code-runner/molten-nvim/init.lua:36]: yeah but this block of code literally does nothing 😂 It doesn't install it or anything, just says "make sure this is installed if the user has already installed it."

Comment (Uzaaft) [reply lua/astrocommunity/code-runner/molten-nvim/init.lua]: Ah I thought it said: Make sure to load this if it's installed.

Comment (mehalter) [reply lua/astrocommunity/code-runner/molten-nvim/init.lua]: Ah, if it were a `dependencies` section then it would be. A lot of times that's not necessary when it's library based because you use things in Lua by requiring the modules and in lazy.nvim we automatically get lazy loading based on module requiring

Comment (Uzaaft) [reply lua/astrocommunity/code-runner/molten-nvim/init.lua]: Got it. Gratzie Mille Signore

**Comment by Uzaaft** [comment]
@mehalter  Messa need a review from youssa

**Comment by mehalter** [comment]
@Uzaaft you should be requesting reviews from @AstroNvim/astrocommunity-maintainers . Not me specifically

**Comment by Uzaaft** [comment]
> @Uzaaft you should be requesting reviews from @AstroNvim/astrocommunity-maintainers . Not me specifically

image

**Review by mehalter — APPROVED** [review]

**Pull request merged**
